vkdgen.py

```diff
@@ -113,7 +113,9 @@ def convertTypeArray(typ, name):
 class DGenerator(OutputGenerator):
 	def __init__(self, errFile=sys.stderr, warnFile=sys.stderr, diagFile=sys.stderr):
 		super().__init__(errFile, warnFile, diagFile)
-	
+		self.enumConstants = []
+		self.funcNames = []
+

```

Comment:
I hope you are talking about your bindings, and not bindings in general, as that opinion is exactly the opposite of mine. A binding should be as close as possible in structure AND readability to the original. Me as D user am not interested in having to search for C headers to understand the coresponding D module.
And now the best part, my pull requests show that it is possible and there is no good argument against it. It does not harm functionality or speed. Its just the proper and cleaner way to do.